content/publish/curvespace.md

```diff
@@ -0,0 +1,16 @@
+---
+title: curve.space
+subject: MyST Websites
+---
+
+Using the Curvenote CLI to deploy your MyST website is the fastest and easiest way to get your content online. [curve.space](https://curve.space) is a free dedicated hosting service for MyST websites that is linked with your account on https://curvenote.com.
+
+You can deploy multiple websites to curve.space to subdomains of your choice either at `https://username.curve.space` or `https://username-any_subdomain_suffix.curve.space` and your content will be made live immedaitely and served using fast, modern web hosting infrastructure backed by Vercel and Google Cloud.
+
+Use `curve.space` to publish academic websites, project documentation or reports, preprints, or anything else that will fit into a MyST Website. Some examples of open technical content published on `curve.space` by our users are:
+
+- [ClimaSoMa Report](https://climasoma.curve.space/) - a multi author report on EU funding
+- [PhD thesis - Geophysical Inversion](https://https://phd.row1.ca/)
+- [PrePrint - La Palma Eruption](https://stevejpurves-lapalma.curve.space/la-palma-earthquakes)
```

Comment:
@rowanc1 last week I read something that your wrote (was it the scipy blogpost?) that had a great list of examples but can't remember where exactly...

Reply:
wasn't the news item...